### PR TITLE
[Release] Linux/SQL Board

### PR DESCRIPTION
**Pull request by minhphuong1994**
finished tickets:
1. [architecture and demo](https://www.notion.so/jarvisdev/Architecture-Demo-ffbd4fba79434253a30f0317a66b1346) 
2. [PSQL&Docker&Gitflow](https://www.notion.so/jarvisdev/PSQL-Docker-GitFlow-852c885bfc2444b5bd87399e9658fc86)
3. [Create DDls](https://www.notion.so/jarvisdev/Creating-DDL-1d6a669d77f84e3290a249cfd5501d9e)

### CODE REVIEW

File: linux_sql/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (jarviscanada) [inline linux_sql/.gitignore:1]: git rm linux_sql/.gitignore

File: linux_sql/scripts/psql_docker.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	;;
+
+	*)
+	echo 'Illigal command'
+	echo 'Commands: start|stop|create'
+	exit 1
+	;;
+esac
+
+exit 0
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh:61]: remove

**Review by jarviscanada — APPROVED** [review]
LGTM

File: linux_sql/.gitignore (hunk 1 above)
Comment (minhphuong1994) [reply linux_sql/.gitignore]: done

**Pull request merged**